### PR TITLE
chore: follow-up 6d1aa4d

### PR DESCRIPTION
**Pull request by BynaryCobweb**

### CODE REVIEW

File: src/backends/torch/torchlib.cc [hunk 1]
```diff
@@ -375,6 +375,7 @@ namespace dd
   int TorchLib<TInputConnectorStrategy, TOutputConnectorStrategy,
                TMLModel>::train(const APIData &ad, APIData &out)
   {
+    using namespace std::chrono;
```
Comment (sileht) [inline src/backends/torch/torchlib.cc:378]: left over ?

Comment (sileht) [reply src/backends/torch/torchlib.cc:378]: Ok you just move it, I think `using namespace` should be forbidden :p

Comment (BynaryCobweb) [reply src/backends/torch/torchlib.cc:378]: was there before. Its scope is restricted to only this function, to shorten calls to system_clock etc

Comment (sileht) [reply src/backends/torch/torchlib.cc:378]: I think it's better to use `using std::chrono::system_lock`, then, no?

**Pull request merged**